internal/lib/sandbox/sandbox.go

```diff
@@ -320,6 +393,111 @@ func (s *Sandbox) RemoveContainer(ctx context.Context, c *oci.Container) {
 	s.containers.Delete(c.Name())
 }
 
+// SetName can be used to set the name of the sandbox.
+func (s *Sandbox) SetID(id string) {
```

Comment:
do you think it makes sense to use `internal/factory/sandbox` package for these? My thought is it can be the translation package between a spec or a cri request, and define all of these setters. then, the sandbox object we pass around doesn't have these setters we only need on initialization. thoughts?

Reply:
We can utilize `SetNameAndID` and set them from there, I'll change it as suggested.